telco-hub/configuration/reference-crs/required/gitops/gitopsSubscription.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-gitops-operator
+  namespace: openshift-gitops-operator
+spec:
+  channel: gitops-1.12
```

Comment:
The current main branch is aligned with OCP 4.18. For gitops operator I believe that the aligned version would be 1.14

Reply:
I suggest, when ztp-site-generate 4.18 is out, we can run the script `get_ztp_installation.sh`  with 4.18 version, and this is updated. 
If I change it now to gitops-1.14, it will require an 4.18 hub, that is something we cannot create now. So, it will fail the deployment tests. 
I could update it to 4.17, would be oka?

Reply:
I think we should have this branch with 4.17, or we will not be able to test if our manifest work. 
But I have changed to 1.13, that is actually the one for ZTP 4.17.
My guess is that, we can update the branch when 4.18 is out, no?

Reply:
Yes, for now aligning with 4.17 for testing is fine. Thanks!